debezium-core/src/main/java/io/debezium/relational/HistorizedRelationalDatabaseConnectorConfig.java

```diff
@@ -58,6 +62,17 @@ public abstract class HistorizedRelationalDatabaseConnectorConfig extends Relati
                     + SchemaHistory.CONFIGURATION_FIELD_PREFIX_STRING + "' string.")
             .withDefault(DEFAULT_SCHEMA_HISTORY);
 
+    public static final Field MAX_RETRIES_ON_ERROR = Field.create(ERRORS_MAX_RETRIES)
+            .withDisplayName("The maximum number of retries")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 2))
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.LOW)
+            .withDefault(DEFAULT_MAX_RETRIES)
+            .withValidation(Field::isInteger)
+            .withDescription(
+                    "The maximum number of retries on connection errors before failing (-1 = no limit, 0 = disabled, > 0 = num of retries).");
+
```

Comment:
If we're going to pull this up to be used by all connectors, then we should put this all in `CommonConnectorConfig` if its used across all relational and non-relational connectors, otherwise only in `RelationalConnectorConfig` if its applicable to relational ones.  

This doesn't seem related to only History-based connectors (SQL Server, Oracle, and MySQL), so this doesn't seem like the right logical place for this.

Reply:
@Naros 
i move this to CommonConnectorConfig plz check